include/rest/api_client.hpp

```diff
@@ -130,8 +129,10 @@ namespace disccord
 
                     pplx::task<void> request_empty(route_info& route, const pplx::cancellation_token& token = pplx::cancellation_token::none())
                     {
-                        set_content_length(0);
-                        return request_internal(route, token).then([](web::http::http_response response){});
+                        disccord::api::request_info* info = new disccord::api::request_info();
+                        
+                        info->set_body("", "application/json");
```

Comment:
In theory you *shouldn't* need this here; since you're sending a simple GET request, there shouldn't be any need to specify any of these headers.